README.md

```diff
@@ -669,6 +669,28 @@ end
 
 If you have defined an action-specific method on your policy for the current action, the `permitted_attributes` helper will call it instead of calling `permitted_attributes` on your controller.
 
+If you have a different way of fetching the parameters other than the standard way:
+
+```ruby
+def pundit_params_for(record)
+  params.require(PolicyFinder.new(record).param_key)
+end
+```
+
+You can override it this method like this:
+
+```ruby
+# If you don't want to use require
+def pundit_params_for(record)
+  params.fetch(PolicyFinder.new(record).param_key, {})
```

Comment:
Does fetch still return the Rails controller specific params implementation? @pablocrivella

Reply:
Yeah, I didn't think this would work actually. According to the docs: http://edgeapi.rubyonrails.org/classes/ActionController/Parameters.html#method-i-fetch it should return the value given to fetch. However, when looking at the code you can clearly see that the value is converted to an instance of `ActionController::Parameters` (if possible). So I guess it should work :)